<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_itemClass</class>
 <widget class="QWidget" name="project_itemClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>project_item</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>所属线路：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>最后修改：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lineName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_lastChange">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>60</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_sel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>30</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_projectName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>1750</x>
     <y>30</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border:0px;
background-color: rgba(255, 255, 255,0);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>1800</x>
     <y>30</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border:0px;
background-color: rgba(255, 255, 255,0);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_createTime">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>创建时间：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_picCount">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>场景数量：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_frontCover">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_see">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>30</y>
     <width>50</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
border:0px;
background-color: rgba(255, 255, 255,0);
font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>浏览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;background-color: rgb(250, 250, 250);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>pushButton_bg</zorder>
  <zorder>label_4</zorder>
  <zorder>label_3</zorder>
  <zorder>label_lineName</zorder>
  <zorder>label_lastChange</zorder>
  <zorder>checkBox_sel</zorder>
  <zorder>label_projectName</zorder>
  <zorder>pushButton_edit</zorder>
  <zorder>pushButton_del</zorder>
  <zorder>label_createTime</zorder>
  <zorder>label_5</zorder>
  <zorder>label_picCount</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_frontCover</zorder>
  <zorder>pushButton_see</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
